src/Tests/API/APIApprovals.ApproveAzureServiceBusTransport.approved.txt

```diff
@@ -479,12 +479,22 @@ namespace NServiceBus.AzureServiceBus
         public System.Threading.Tasks.Task Send(Microsoft.ServiceBus.Messaging.BrokeredMessage message) { }
         public System.Threading.Tasks.Task SendBatch(System.Collections.Generic.IEnumerable<Microsoft.ServiceBus.Messaging.BrokeredMessage> messages) { }
     }
-    public class NamespaceInfo
+    public class NamespaceDefinition : System.IEquatable<NServiceBus.AzureServiceBus.NamespaceDefinition>
```

Comment:
I see, one is at configuration time, the other at runtime. Maybe we could go for different names here to better reflect that (NamespaceConfiguration and NamespaceInstance?), thoughts @SeanFeldman ?